server/webhook_parser.go

```diff
@@ -403,6 +405,16 @@ func preProcessText(jiraMarkdownString string) string {
 		return "> " + quotedText
 	})
 
+	processedString = noFormatRegex.ReplaceAllStringFunc(processedString, func(noFormatBlock string) string {
```

Comment:
Can we add a comment above this?

Reply:
function header has a brief comment about the functionality. Adding comment on regex will be a bit extra